Packs/QRadar/TestPlaybooks/playbook-TestQradarV2.yml

```diff
@@ -1235,6 +1278,75 @@ tasks:
     ignoreworker: false
     skipunavailable: false
     quietmode: 0
+  "46":
+    id: "46"
+    taskid: 4a63afb9-f035-4579-8a9c-7f3368276e2b
+    type: condition
+    task:
+      id: 4a63afb9-f035-4579-8a9c-7f3368276e2b
+      version: -1
+      name: Check name is ACF2 rule key
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "yes":
+      - "47"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: QRadar.Properties.name
+                filters:
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: QRadar.Properties.name
+                      iscontext: true
+                    right:
+                      value:
+                        simple: ACF2 rule key
```

Comment:
isnt risky to set here hard coded value and check for equality ? will be property always be in our instance?

Reply:
We must use a key that we have in the instance. I do not think it'll vanish anyway.